tests/webview/ui/test_content.py

```diff
@@ -163,6 +163,22 @@ def test_get_this_book(webview_base_url, selenium):
     assert offline_zip_displayed == downloads.is_offline_zip_available
 
 
+@markers.webview
+@markers.nondestructive
+def test_section_title(webview_base_url, selenium):
+    # GIVEN the home page and a book
+    home = Home(selenium, webview_base_url).open()
+    book = home.featured_books.openstax_list[0]
+
+    # WHEN the book's cover is clicked
+    content = book.click_book_cover()
+
+    # THEN the section title does not contain HTML markup
+    section_title = content.section_title
+    assert '<' not in section_title
+    assert '>' not in section_title
+
+
 @markers.webview
 @markers.nondestructive
 def test_content(webview_base_url, selenium):
```

Comment:
This also looks like it needs a more specific name and testrail marker

Reply:
oh snap, that's from the old import where i didn't include that, my badz. @kerwinso can you help dante locate the case id for this one?